crates/optimism/rpc/src/eth/proofs.rs

```diff
@@ -5,14 +5,14 @@ use alloy_primitives::Address;
 use alloy_rpc_types_eth::EIP1186AccountProofResponse;
 use alloy_serde::JsonStorageKey;
 use async_trait::async_trait;
-use derive_more::Constructor;
 use jsonrpsee::proc_macros::rpc;
 use jsonrpsee_core::RpcResult;
 use jsonrpsee_types::error::ErrorObject;
-use reth_optimism_trie::{provider::OpProofsStateProviderRef, OpProofsStorage, OpProofsStore};
-use reth_provider::{BlockIdReader, ProviderError, ProviderResult, StateProofProvider};
+use reth_optimism_trie::{OpProofsStorage, OpProofsStore};
+use reth_provider::StateProofProvider;
 use reth_rpc_api::eth::helpers::FullEthApi;
-use reth_rpc_eth_types::EthApiError;
+
```

Comment:
```suggestion
```
let's stick to this way of formatting module import prelude to make it easy to upstream